### PR TITLE
CRSF docs, some adjustments

### PR DESCRIPTION
**Pull request by olliw42**
some changes to the CRSF docs page
main purpose is to remove the various suggestions for teh baudrates, as mLRS shouls work with the defaults, and to rather move them to suggestion for optimization. 
Also the github tools for highlights is used, and some changes ehere and where in language.

### CODE REVIEW

File: docs/CRSF.md [hunk 1]
```diff
@@ -14,7 +14,10 @@ Optional but recommended steps:
 - Set the receiver into "MAVLink mode" (described below).
 - Install the Yaapu app (described below).
 
-The mLRS default parameter settings follow the below recommendations. That is, with a freshly flashed mLRS Tx module and receiver all you need to do is to set up your radio and flight controller!
+> [!TIP]
+> The mLRS default parameter settings follow the below recommendations. That is, with a freshly flashed mLRS Tx module and receiver all you need to do is to set up your radio and flight controller.
+
+It is recommeneded that for first setup the Tx module and receiver are left in their default configuration. Once your system is up and running there is still time to explore the options for optimizing the performacne.
```
Comment (jlpoltrack) [inline docs/CRSF.md:20]: Perhaps 'It is recommended for the initial setup the Tx module and receiver are left in their default configuration. Once your system is up and running then you can explore the options for further optimizing performance.'

File: docs/CRSF.md [hunk 2]
```diff
@@ -26,41 +29,43 @@ The mLRS default parameter settings follow the below recommendations. That is, w
...
-    - 57 (= 57600) for 31 Hz
-    - 38 (= 38400) for 19 Hz (57 works very well too, only parameter download is slower with ArduPilot 4.5)
-    - 230 (= 230400) for FLRC
+- SERIALx_BAUD: 57 (= 57600)
 - SERIALx_PROTOCOL = 2 (important, do not use MAVLink v1!)
 - SERIALx_OPTIONS = 4096 (ignore commands from GCS to change stream rates)
 
 ***Note***: 'x' refers to the serial port of your flight controller used for connecting with the mLRS receiver's serial port.
 
+The baud rate setting in SERIALx_BAUD is not critical, as mLRS will gracefully work with them all. However, depending on the ArduPilot firmware version, parameter download can be optimized somewhat by "tuning" the baudrate. For instance, one can try:
```
Comment (jlpoltrack) [inline docs/CRSF.md:63]: Perhaps, 'However, for ArduPilot version 4.5 or lower, parameter download...'

**Comment by jlpoltrack** [comment]
Few comments, otherwise looks good.

File: docs/CRSF.md (hunk 1 above)
Comment (olliw42) [reply docs/CRSF.md:20]: done

File: docs/CRSF.md [hunk 3]
```diff
@@ -26,41 +29,43 @@ The mLRS default parameter settings follow the below recommendations. That is, w
...
+> [!TIP]
+> The mLRS default settings are Tx Ch Source = crsf, Tx Ser Baudrate = 115200, Tx Ser Dest = serial, Tx Snd RadioStat = on. Therefore, except of Tx Ser Dest, adjustment of the parameters is usually not needed.
 
-The mLRS default settings are Tx Ch Source = crsf, Tx Ser Baudrate = 115200, Tx Ser Dest = serial, Tx Snd RadioStat = on.
+***Notes***: 
+- Tx Ser Baudrate should be larger than the link data rate in order to provide enough capacity (e.g., in the 50 Hz mode the link data rate is 4100 Bytes/sec and the baudrate should thus be larger than 41000), but otherwise the choice is not critical and largely determined by the user's need. The default value is 115200, and is a good choice for most cases.
+- While not necessary, for the FLRC mode it can be benefitial to use 230400.
```
[the remark this answers is not in the document]
Comment (olliw42) [reply docs/CRSF.md:49]: done

File: docs/CRSF.md (hunk 2 above)
Comment (olliw42) [reply docs/CRSF.md:63]: done

**Comment by olliw42** [comment]
THX. changes done.

**Pull request merged**